src/parser/cvc/Cvc.g

```diff
@@ -2207,18 +2200,19 @@ simpleTerm[CVC4::api::Term& f]
       std::stringstream strRat;
       strRat << r;
       f = SOLVER->mkReal(strRat.str());
+      // This is no longer required
```

Comment:
If it's no longer required, it should be removed.

Reply:
will do